scripts/scripts.js

```diff
@@ -217,4 +224,14 @@ async function loadPage() {
   loadDelayed();
 }
 
+export function decorateInnerHrefButtonsWithArrowIcon(block) {
+  block.querySelectorAll('a').forEach((a) => {
+    const icon = domEl('calcite-icon', { icon: 'arrowRight', scale: 's' });
+    icon.style.marginInlineStart = '4px';
+    if (a.href.includes('esri.com') || (a.href.includes('main--esri'))) {
```

Comment:
I did use that example, but can you make sure to use "--esri--aemsites.aem."? My point is that this should work for esri sites across all branches in any environment